src/Trace/Propagator/JaegerPropagator.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * Copyright 2017 OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// reference: https://www.jaegertracing.io/docs/1.18/client-libraries/#propagation-format
+
+
+namespace OpenCensus\Trace\Propagator;
+
+use OpenCensus\Trace\SpanContext;
+
+/**
+ * This propagator uses HTTP headers to propagate SpanContext over HTTP.
+ * The default headers is `uber-trace-id`.
+ */
+class JaegerPropagator implements PropagatorInterface
```

Comment:
Instead, there should be jaeger impl for FormatterInterface, used with HttpHeaderPropagator :)

Reply:
agreed